.ci/update-ide-properties.sh

```diff
@@ -1,3 +1,7 @@
+# Install AWS CLI
```

Comment:
`echo`?

Reply:
Not required, since we already have an echo before the script is executed.